app/models/recipe.rb

```diff
@@ -11,5 +11,7 @@ class Recipe < ApplicationRecord
   }
   validates :difficulty, presence: true
   validates :name, presence: true
+
+  scope :recipe_is_favorited, -> { where(favorite: true) }
```

Comment:
For the name of this scope I don't think we need to call it `recipe_is_favorited` when we use the scope from the `User` model it will look like `user.recipes.recipe_is_favorited` which reads kind of strange. 

I think changing the name of the scope to something a little more simplified like `favorites` or `favorited` may read better ie. `user.recipes.favorites` or `user.recipes.favorited`.